github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
this is the current list: https://github.com/dataxu/github-pr/blob/master/github_pr.py#L26
```
table_headers = ["#", "State", "Status", "Merge", "Base", "Head", "Title"]
```